soccer/MainWindow.hpp

```diff
@@ -204,4 +208,6 @@ private Q_SLOTS:
     std::vector<QPushButton*> _logPlaybackButtons;
 
     RJ::Time _firstLogTimestamp = -1;
+
+    Packet::SimCommand quickLoadCmd;
```

Comment:
Rename this to `_quickLoadCmd` to match the naming conventions with the other instance variables.